.circleci/config.yml

```diff
@@ -145,12 +145,13 @@ workflows:
               only:
                 - main
       - deploy-snapshot:
-          requires:
-            - assemble
+#          requires:
+#            - assemble
```

Comment:
I'm assuming this bit and line 154 were for testing?

Reply:
yes, I am still testing some stuff